src/apps/persistence/entities.clj

```diff
@@ -50,6 +50,9 @@
 ;; References associated with an app.
 (defentity app_references)
 
+;; Extra info for HTCondor
+(defentity apps_htcondor_extra)
```

Comment:
We probably don't need this `defentity` defined, since we're not depending on korma to join this table in any queries. So `get-app-extra-info` could select using a keyword instead: `select :apps_htcondor_extra (fields ...`

It also doesn't hurt to leave it defined, now that it's already added 👍

Reply:
Yeah, I weighed looking up how to do it with the keyword and just figured it was easier to add them into this namespace. Maybe at some point we'll standardize, but I don't think this is any worse than our existing split, heh.